docs/maya/activation.md

```diff
@@ -0,0 +1,78 @@
+# AdnActivation
+
+The AdnActivation node is a dependency node that allows operations on activation values for the computation of a final activation value that can be used to drive, for example, muscle activations. An example of its use case could be the merging of various sensor activations into one single driven activation to activate a muscle given multiple poses.
+
+## How To Use
+
+To create this node, follow these steps:
+
+1. Open the Node Editor from Windows > Node Editor.
+2. Press the Tab key, type *AdnActivation* and press Enter to create a new instance of this node.
+
+The activation node can now be used to override, add, subtract, multiply or divide the activations from different sources (sensors) into one final activation value.
+For example, multiple distance sensors of a character can be merged together to produce different kinds of activations throughout the simulation.
+
+<figure markdown>
+  ![AdnActivation example](images/adn_activation_create.png)
+  <figcaption><b>Figure 1</b>: Use case in which AdnActivation is created for merging two activations to drive a muscle's activation.</figcaption>
+</figure>
+
+To add new inputs to the AdnActivation node:
+
+1. Go to the AET and press *Add New Item*.
+2. Set the desired *Operator*.
+3. Type in a fixed value in *Value*, animated it or connect the plug with an AdonisFX sensor's output.
+
+> [!NOTE]
+> All operators will be evaluated from top to bottom (starting from index 0 and ending on index N).
+
+## Operators
+
+1. **Over (Override)**: Overrides the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0 the output activation will be 2.0.
+2. **Add (Add)**: Adds the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0 the output activation will be 3.0.
+3. **Sub (Subtract)**: Subtracts the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0 the output activation will be 0.0.
```

Comment:
This really depends on the output min value. I think we should add rewrite this a little:
```suggestion
3. **Sub (Subtract)**: Subtracts the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0, then new accumulated values will be -1.0.
```

Using "the new accumulated" we are not referringto the final value. We will have to add another paragraph or a Note box saying: the final value will be clamped based on the min and max output values

Reply:
Done!